tests/util/test_full_block_utils.py

```diff
@@ -129,20 +131,21 @@ def get_foliage_transaction_block():
     )
 
 
-def get_transactions_info(height):
-    yield None
+def get_transactions_info(height, foliage_transaction_block):
```

Comment:
I realize I failed to add type annotations to this file when I wrote it, but I think we should work towards adding that everywhere, starting with new code we add.

is `foliage_transaction_block` a `bool`? because it doesn't look like what's passed in to this call is.

Reply:
i'll do this